uvloop/sslproto.pxd

```diff
@@ -6,6 +6,20 @@ cdef enum SSLProtocolState:
     SHUTDOWN = 4
 
 
+cdef enum AppProtocolState:
+    # This tracks the state of app protocol (https://git.io/fj59P):
+    #     start -> CM [-> DR*] [-> ER?] -> CL -> end
+    # * CM: connection_made()
+    # * DR: data_received()
+    # * ER: eof_received()
+    # * CL: connection_lost()
+
+    START = 0
+    AFTER_CM = 1
+    AFTER_ER = 2
```

Comment:
Can we rename these to `STATE_INIT`, `STATE_CON_MADE`, `STATE_EOF`, `STATE_CON_LOST`? Also the comment isn't synced with enum members.

Reply:
Makes sense to me! Updated with a state diagram, and fixed the comment too.